### PR TITLE
OSSM-6044 [DOC] Update Kiali CR and change name-space to istio-system

### PR DESCRIPTION
**Pull request by gwynnemonahan**
[OSSM-6044](https://issues.redhat.com//browse/OSSM-6044) [DOC] Update Kiali CR and change name-space to istio-system

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSSM-6044

Link to docs preview:
https://72765--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-observability#ossm-integrating-with-user-workload-monitoring_observability

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/ossm-integrating-with-user-workload-monitoring.adoc [hunk 1]
```diff
@@ -45,19 +45,23 @@ apiVersion: kiali.io/v1alpha1
...
-    prometheus:
-      auth:
-        token: secret:thanos-querier-web-token:token
+spec: 
+  external_services: 
+    istio:
+      config_map_name: istio-<smcp-name>
+      istio_sidecar_injector_config_map_name: istio-sidecar-injector-<smcp-name>
+      istiod_deployment_name: istiod-<smcp-name>
+      url_service_version: 'http://istiod-<smcp-name>.istio-system:15014/version'
```
Comment (jewertow) [inline modules/ossm-integrating-with-user-workload-monitoring.adoc:54]:
```suggestion
```

File: modules/ossm-integrating-with-user-workload-monitoring.adoc [hunk 2]
```diff
@@ -45,19 +45,18 @@ apiVersion: kiali.io/v1alpha1
...
+      auth: 
         type: bearer
-        use_kiali_token: false
+        use_kiali_token: true
       query_scope:
         mesh_id: "basic-istio-system"
       thanos_proxy:
         enabled: true
       url: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
+  version: v1.65
```
Comment (jewertow) [inline modules/ossm-integrating-with-user-workload-monitoring.adoc:59]:
```suggestion
```

File: modules/ossm-integrating-with-user-workload-monitoring.adoc [hunk 3]
```diff
@@ -45,19 +45,18 @@ apiVersion: kiali.io/v1alpha1
 kind: Kiali
 metadata:
   name: kiali-user-workload-monitoring
-  namespace: istio-system
```
Comment (jewertow) [inline modules/ossm-integrating-with-user-workload-monitoring.adoc:48]:
```suggestion
  namespace: istio-system
```

**Review by jewertow — COMMENTED** [review]
LGTM

**Review by pbajjuri20 — APPROVED** [review]

**Review by kowen-rh — APPROVED** [review]
LGTM! :+1: 

/remove-label peer-review-in-progress
/label peer-review-done

**Pull request merged**